grobid-trainer/resources/dataset/name/citation/corpus/crossref_raw_citations.xml

```diff
@@ -0,0 +1,21 @@
+
+<author><lastname>Castillo</lastname>, <forename>R</forename>. <forename>E</forename>., <lastname>Castro</lastname>, <forename>P</forename>. <forename>J</forename>. <forename>M</forename>., <lastname>Cayabyab</lastname>, <forename>G</forename>. <forename>T</forename>., and <lastname>Rachel Aton</lastname>, <forename>M</forename>.</author>
+<author><lastname>Lilford</lastname> <forename>R</forename>, <lastname>Hughes</lastname> <forename>E</forename>, <lastname>Vandekerckhove</lastname> <forename>P</forename></author>
+<author><forename>Ben</forename> <lastname>Taskar</lastname> <forename>Pieter</forename> <lastname>Abbeel</lastname> and <forename>Daphne</forename> <lastname>Koller</lastname></author>
+<author><forename>Meng</forename> <lastname>Qu</lastname> and <forename>Jian</forename> <lastname>Tang</lastname></author>
+<author><forename>Bishan</forename> <lastname>Yang</lastname> <forename>Scott</forename> <forename>Wen-tau</forename> <lastname>Yih</lastname> <forename>Xiaodong</forename> <lastname>He</lastname> <forename>Jianfeng</forename> <lastname>Gao</lastname> and <forename>Li</forename> <lastname>Deng</lastname></author>
+<author><forename>Günter</forename> <lastname>Klambauer</lastname> <forename>Thomas</forename> <lastname>Unterthiner</lastname> <forename>Andreas</forename> <lastname>Mayr</lastname> and <forename>Sepp</forename> <lastname>Hochreiter</lastname></author>
+<author><forename>Byung-Hak</forename> <lastname>Kim</lastname> <forename>Ethan</forename> <lastname>Vizitei</lastname> and <forename>Varun</forename> <lastname>Ganapathi</lastname></author>
+<author><forename>Sercan</forename> <forename>O</forename>. <lastname>Arik</lastname> and <forename>Tomas</forename> <lastname>Pfister</lastname></author>
+<author><forename>Mohammad</forename> <lastname>Al Hasan</lastname>, <forename>Vineet</forename> <lastname>Chaoji</lastname>, <forename>Saeed</forename> <lastname>Salem</lastname>, and <forename>Mohammed</forename> <lastname>Zaki</lastname></author>
+<author><forename>Johan</forename> <forename>J</forename> <lastname>Vossensteyn</lastname> <forename>Andrea</forename> <lastname>Kottmann</lastname> <forename>Benjamin</forename> <forename>WA</forename> <lastname>Jongbloed</lastname> <forename>Franciscus</forename> <lastname>Kaiser</lastname> <forename>Leon</forename> <lastname>Cremonini</lastname> <forename>Bjorn</forename> <lastname>Stensaker</lastname> <forename>Elisabeth</forename> <lastname>Hovdhaugen</lastname> and <forename>Sabine</forename> <lastname>Wollscheid</lastname></author>
+<author><forename>Xue</forename> <forename>Bin</forename> <lastname>Peng</lastname>, <forename>Marcin</forename> <lastname>Andrychowicz</lastname>, <forename>Wojciech</forename> <lastname>Zaremba</lastname>, and <forename>Pieter</forename> <lastname>Abbeel</lastname></author>
```

Comment:
Hard to say if Bin is a middle name or a forename (composed forename "Xue Bin").

Reply:
From ORCiD and other sources, looks like composed ("Xue Bin" or "Xue-Bin"), I will make it a single forename tag around both parts.